java-client/build.gradle.kts

```diff
@@ -123,7 +123,7 @@ val integrationTest = task<Test>("integrationTest") {
 
 dependencies {
 
-    val opensearchVersion = "2.0.0-alpha1-SNAPSHOT"
+    val opensearchVersion = "2.0.0"
     val jacksonVersion = "2.12.6"
```

Comment:
Please update Jakson as well [1]:

```
val jacksonVersion = "2.13.2"
val jacksonDatabindVersion = "2.13.2.2"
```

[1] https://github.com/opensearch-project/OpenSearch/blob/2.0/buildSrc/version.properties#L12

Reply:
Sure! Updated the jackson versions.

Reply:
:+1:

Reply:
@reta I recall you had a suggestion(via gradle) to import the versions from opensearch. 
Is it something we could use here?

Reply:
@saratvemulapalli that would be great but we haven't done any work towards that yet (afaik)

Reply:
Opened up an issue: https://github.com/opensearch-project/opensearch-java/issues/161
Not sure if there is one on OpenSearch, if not I'll open one.